src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerResolverTest.php

```diff
@@ -11,15 +11,16 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Tests\Controller;
 
+use Psr\Container\ContainerInterface as Psr11ContainerInterface;
 use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\ControllerNameParser;
 use Symfony\Bundle\FrameworkBundle\Controller\ControllerResolver;
 use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpKernel\Tests\Controller\ControllerResolverTest as BaseControllerResolverTest;
+use Symfony\Component\HttpKernel\Tests\Controller\Psr11ControllerResolverTest;
 
-class ControllerResolverTest extends BaseControllerResolverTest
+class ControllerResolverTest extends Psr11ControllerResolverTest
```

Comment:
The tests cases were simply moved in the `Psr11ControllerResolverTest` class and are inherited, so I think we're safe, aren't we? :)